codemods/remove-cozy-theme.js

```diff
@@ -0,0 +1,42 @@
+// Press ctrl+space for code completion
+
+const imports = require('@cozy/codemods/src/imports')
+
+const strip = txt => txt.replace(/\n/g, '').replace(/\s/g, '')
+const nonEmpty = node => {
+  if (
+    (node.type === 'Literal' || node.type == 'JSXText') &&
+    strip(node.value) === ''
+  ) {
+    return false
+  }
+  return true
+}
+
+module.exports = function transformer(file, api) {
+  const j = api.jscodeshift
+  const root = j(file.source)
+
+  root
+    .find(j.JSXElement, {
+      openingElement: { name: { name: 'MuiCozyTheme' } }
+    })
+    .forEach(path => {
+      const nonEmptyChildren = path.node.children.filter(nonEmpty)
+      if (nonEmptyChildren.length == 1) {
+        path.replace(nonEmptyChildren[0])
+      } else {
+        path.replace(
+          j.jsxFragment(
+            j.jsxOpeningFragment(),
+            j.jsxClosingFragment(),
+            nonEmptyChildren
+          )
+        )
+      }
+    })
+
+  imports.removeUnused(root)
+
+  return root.toSource()
+}
```

Comment:
Didn't we say that the codemods were in @cozy/codemods?

Reply:
I am not sure that it makes sense here as I do not expect this codemod to be shared.